src/main/scala/com/evojam/play/elastic4s/client/ElasticSearchClient.scala

```diff
@@ -4,10 +4,14 @@ import scala.concurrent.{ExecutionContext, Future}
 
 import play.api.libs.json.Writes
 
+import com.sksamuel.elastic4s.SearchDefinition
+
+import com.evojam.play.elastic4s.core.search.PreparedSearch
+
 trait ElasticSearchClient {
 
   /**
-   * Index a document
+   * Index or update a document
```

Comment:
For clarity maybe write exactly under what conditions either index or update happens